README.md

```diff
@@ -1,5 +1,55 @@
 # Actions workflow run
 
+This GitHub Action will allow you to easily execute Chassy workflows within
+your automation pipelines.
+
+# Environment
```

Comment:
Nit: Maybe rename to Authentication with Chassy or something along the lines of that. Since it is in effect talking about how we have chosen to implement authentication with Chassy. We have a feature called environment that could potentially confuse readers.

Reply:
renamed to *Authentication with Chassy*